### PR TITLE
Seperate the ppc64_cpu tests

### PR DESCRIPTION
**Pull request by narasimhan-v**
Seperated the ppc64_cpu tests into test_params and test_smt_loop.

Signed-off-by: Narasimhan V <sim@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
```
 (1/2) ppc64_cpu_test.py:PPC64Test.test_params: PASS (20.35 s)
 (2/2) ppc64_cpu_test.py:PPC64Test.test_smt_loop: PASS (18.93 s)
```

**Comment by narasimhan-v** [comment]
@sathnaga I will take `test_cmd_options`

File: cpu/ppc64_cpu_test.py [hunk 1]
```diff
@@ -172,20 +168,21 @@ def dscr(self):
...
-    @staticmethod
-    def smt_loop():
+    def test_smt_loop(self):
         """
         Tests smt on/off in a loop
         """
         for _ in range(1, 100):
-            process.run("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
-                        shell=True)
+            if process.system("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
```
Comment (nasastry) [inline cpu/ppc64_cpu_test.py]: `ppc64_cpu --smt=` is used in many places for better maintainability of code please use a variable.

**Comment by narasimhan-v** [comment]
[ppc64_cpu_job_log.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/2134130/ppc64_cpu_job_log.txt)

File: cpu/ppc64_cpu_test.py (hunk 1 above)
Comment (narasimhan-v) [reply cpu/ppc64_cpu_test.py]: @nasastry Thanks. Done, please have a check now.

**Comment by PraveenPenguin** [comment]
@sathnaga @nasastry thanks for review

**Pull request merged**